Shopstagram - rozwiązanie/css/components/page-header/_page-header.scss

```diff
@@ -14,11 +15,17 @@
     &__logo {
         flex-shrink: 0;
         
-        img {
-            display: block;
-        }
+            img {
+              display: block;
+            }
+            
     }
 
+    // Christmas logo
+
+    // @include addChristmasLogo;
+    @include themeLogo (30, 30, -4, 120, "tree.png");
```

Comment:
jesli dodalas ten mixin tutaj, czyli defakto wkleilas tu style themowe to one sie wyswietla na normalnej stronie tez, a chcemy zeby to dzialalo tak ze jak podpinasz plik _christmas to sa style swiateczne, a jak go odepniemy to one znikna